### PR TITLE
Census Household increase test coverage of downstream data

### PR DESCRIPTION
**Pull request by yoganandkunche**
### What is the context of this PR?

Dumps the answers that are going to be submitted and compares with the answers that were entered for Census_Household

Also updated Census_individual.

### How to review

Run the integration test.

### CODE REVIEW

**Review by jonnyshaw89 — APPROVED** [review]

File: tests/integration/census/test_census_household_submission_data.py [hunk 1]
```diff
@@ -0,0 +1,1144 @@
...
+        self.household_and_accommodation_section()
+
+        self.individual_section()
+
+        self.visitors_section()
+
+        # There are no validation errors (we're on the summary screen)
+        self.assertInPage('You’re ready to submit your 2017 Census Test')
+
+    def visitors_section(self):
```
Comment (collisdigital) [inline tests/integration/census/test_census_household_submission_data.py]: split this into visitor_1 and visitor_2 functions?

File: tests/integration/census/test_census_household_submission_data.py [hunk 2]
```diff
@@ -0,0 +1,1144 @@
...
+            self.post(post_data=post)
+
+        # Completed Visitors sections
+        self.assertInPage('You have completed all questions for Visitor 2')
+        self.post(action='save_continue')
+
+        self.assertInPage('You have successfully completed the ‘Visitors’ section')
+        self.post(action='save_continue')
+
+    def individual_section(self):
```
Comment (collisdigital) [inline tests/integration/census/test_census_household_submission_data.py]: Split this into a function per person

File: tests/integration/census/test_census_individual_submission_data.py [hunk 3]
```diff
@@ -16,183 +16,192 @@ def test_census_individual_data_matches_census_individual(self):
     def get_expected_submission_data():
         expected_downstream_data = [
             {
-                "answer_instance": 0,
+                "group_instance": 0,
```
Comment (collisdigital) [inline tests/integration/census/test_census_individual_submission_data.py:19]: why has this file changed?

File: tests/integration/census/test_census_individual_submission_data.py [hunk 4]
```diff
@@ -537,14 +546,14 @@ def complete_survey(self, eq_id, form_type_id):
...
             {
-                'national-identity-england-answer': ['No religion',
-                                                     'Christian (Including Church of England, Catholic, Protestant and all other Christian denominations)',
-                                                     'Buddhist',
-                                                     'Hindu',
-                                                     'Jewish',
-                                                     'Muslim',
-                                                     'Sikh',
-                                                     'Other'],
+                'religion-answer': ['No religion',
```
Comment (collisdigital) [inline tests/integration/census/test_census_individual_submission_data.py:549]: why the change here?

**Review by collisdigital — CHANGES_REQUESTED** [review]
See comments

File: tests/integration/census/test_census_individual_submission_data.py (hunk 4 above)
Comment (yoganandkunche) [reply tests/integration/census/test_census_individual_submission_data.py:549]: Previously I was wrongly posting the answer on name: 'national-identity-england-answer' instead of 'religion-answer' which I realized when I manually verified the Data second time while working on the census-household test.

File: tests/integration/census/test_census_individual_submission_data.py (hunk 3 above)
Comment (yoganandkunche) [reply tests/integration/census/test_census_individual_submission_data.py:19]: Same as the comment below.

File: tests/integration/census/test_census_household_submission_data.py [hunk 5]
```diff
@@ -0,0 +1,1147 @@
...
+
+    def complete_survey(self, eq_id, form_type_id):
+        variant_flags = {'sexual_identity': 'false'}
+        self.launchSurvey(eq_id, form_type_id, region_code='GB-ENG', variant_flags=variant_flags, roles=['dumper'])
+
+        self.who_live_here_section()
+
+        self.household_and_accommodation_section()
+
+        self.individual_section_Person_1()
```
Comment (ajmaddaford) [inline tests/integration/census/test_census_household_submission_data.py]: Person should be lowercase in the method name

File: tests/integration/census/test_census_household_submission_data.py [hunk 6]
```diff
@@ -0,0 +1,1147 @@
...
+            }
+        ]
+
+        return expected_downstream_data
+
+    def complete_survey(self, eq_id, form_type_id):
+        variant_flags = {'sexual_identity': 'false'}
+        self.launchSurvey(eq_id, form_type_id, region_code='GB-ENG', variant_flags=variant_flags, roles=['dumper'])
+
+        self.who_live_here_section()
```
Comment (ajmaddaford) [inline tests/integration/census/test_census_household_submission_data.py]: I think these method names should be prefixed with `complete_`, so this would be `complete_who_lives_here_section` to make it clearer what the method is doing.
Also, it should be lives rather than live.

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

File: tests/integration/census/test_census_household_submission_data.py (hunk 5 above)
Comment (yoganandkunche) [reply tests/integration/census/test_census_household_submission_data.py]: changed it to 'person'

File: tests/integration/census/test_census_household_submission_data.py (hunk 6 above)
Comment (yoganandkunche) [reply tests/integration/census/test_census_household_submission_data.py]: made the suggested changes

**Review by ajmaddaford — APPROVED** [review]

**Review by collisdigital — APPROVED** [review]
